dragonfly/entity/falling_block.go

```diff
@@ -0,0 +1,98 @@
+package entity
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/entity/physics"
+	"github.com/df-mc/dragonfly/dragonfly/entity/state"
+	"github.com/df-mc/dragonfly/dragonfly/internal/entity_internal"
+	"github.com/df-mc/dragonfly/dragonfly/internal/item_internal"
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"sync/atomic"
+)
+
+// FallingBlock is the entity form of a block that appears when a gravity-affected block loses its support.
+type FallingBlock struct {
+	Block         world.Block
```

Comment:
I wouldn't personally export this field, as the type of a FallingBlock probably shouldn't be changed once set in `NewFallingBlock`.